Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: all install clean uninstall package
 
 export INCLAVARE_CONTAINERS_VERSION := $(shell cat ./VERSION)
-components := rune shim epm sgx-tools
+components := rune shim epm sgx-tools shelter
```

Comment:
It is still unclear that the build failure happens from a scratch environment, such as:

```
CGO_ENABLED=1 go build "-mod=vendor" -buildmode=pie   -ldflags "" -o /root/dev/gopath/src/github.com/alibaba/inclavare-containers/shelter/racommand.o .
# github.com/inclavare-containers/shelter/remoteattestation
/usr/bin/ld: cannot find -lsgx_dcap_quoteverify
collect2: error: ld returned 1 exit status
Makefile:60: recipe for target '/root/dev/gopath/src/github.com/alibaba/inclavare-containers/shelter/racommand.o' failed
```

Reply:
This issue is resolved now.